pkg/services/ngalert/store/alert_rule_state.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+package state;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "./pb";
+
+message AlertInstanceKey {
+    int64 rule_org_id = 1;
+    string rule_uid = 2;
+    string labels_hash = 3;
+}
+
+message AlertInstance {
+    AlertInstanceKey key = 1;
+    map<string, string> labels = 2;
```

Comment:
What about annotations?

Reply:
We don't save them currently, so I didn't add them here

Reply:
Huh, existing AlertInstance does not have it, which I think is a bug! Anyway, let's keep it this way for now